PhoneNumberKit/UI/TextField.swift

```diff
@@ -34,10 +34,12 @@ open class PhoneNumberTextField: UITextField, UITextFieldDelegate {
         super.text = newValue
     }
 
+    private lazy var _defaultRegion: String = PhoneNumberKit.defaultRegionCode()
+
     /// Override region to set a custom region. Automatically uses the default region code.
-    open var defaultRegion = PhoneNumberKit.defaultRegionCode() {
```

Comment:
if you want to override `textField.defaultRegion` in a subclass, why not just @override it the way you want?  these changes seem to change the behavior in breaking ways …

Reply:
because of https://stackoverflow.com/questions/26691935/overriding-a-stored-property-in-swift

Why is the behaviour in breaking ways? `defaultRegion` is used only on initialization so setter is useless.

Reply:
for safety, we can add a setter to `defaultRegion` with a warning that setter is deprecated and unused, and suggest defining a subclass.

Such a variant do not break any existing code.

What do you think about it?